requirements.txt

```diff
@@ -7,4 +7,7 @@ osbrain
 dash
 dash_cytoscape
 networkx
-plotly
\ No newline at end of file
+plotly
+time-series-buffer
+time-series-metadata
+uncertainties
```

Comment:
After merging the associated PR PTB-PSt1/time-series-buffer#4 in time-series-buffer we should now remove this.